Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+# What is an Inclusive Minute
+
+An inclusive minute (or inclusive moment) is an inclusivity practice designed for teams to easily add to existing meeting agendas.
+The practice is intentionally kept small, informal and uncontroversial to maximize the likelihood of adoption and uptake.
+The purpose is to raise awareness, inspire dialog, deeper thinking and/or inform future action.
+Although rather modest in its aims, especially when compared with the need it is intended to address, it is also a useful approach in helping to seed changes in culture.
+
+While a minute is never enough time, we aim to characterize a practice other teams can easily follow and adopt into their existing agendas.
+While we are all impacted by inclusion, sometimes in deeply personal ways, we aim for a focus that relates primarily to a team's existing work activities.
```

Comment:
By inclusion? Or by practices that foster or inhibit inclusion?

Reply:
Good point...not sure how best to re-word...by the lack of inclusion or by issues related to inclusion? Maybe just by inclusion issues

```suggestion
While we are all impacted by inclusion issues, sometimes in deeply personal ways, we aim for a focus that relates primarily to a team's existing work activities.
```

Reply:
That clarifies it entirely for me! `issues` works.